draft-normington-jsonpath-latest.md

```diff
@@ -425,22 +428,25 @@ An array index is a union element consisting of an integer (in base 10).
 ~~~~ abnf
 array-index = integer
 
-integer = [%x2D] (%x30 / (%x31-39 *%x30-39))
+integer = ["-"] ("0" / (%x31-39 *%x30-39))
```

Comment:
You may want to make use of DIGIT and DIGIT1 here

Reply:
Whoops. Of course.